types/blessed/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for blessed 0.1.5
+// Type definitions for blessed 0.1.6
```

Comment:
Can you turn linting on for this file by [adding a `tslint.json`](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/1066f6fb8280aa7bc0553c27ca76739b4666c6bb#lint) and fixing up the header to only include major.minor version numbers?

Something as simple as

```json
{
    "extends": "dtslint/dt.json"
}
```

will work.

Reply:
after add tslint.json and `npm run lint blessed` I get many errors. How should I do?

Reply:
you can add exceptions to rules you do not want to fix. and fix ones you can